Parser/asdl_c.py

```diff
@@ -1117,8 +1118,8 @@ def visitModule(self, mod):
             if (!args_tuple) {
                 goto cleanup;
             }
-            result = Py_BuildValue("ONO", Py_TYPE(self), args_tuple,
-                                   remaining_dict);
+            result = Py_BuildValue("ONN", Py_TYPE(self), args_tuple,
+                                   dict);
```

Comment:
```suggestion
            result = Py_BuildValue("ONN", Py_TYPE(self), args_tuple, dict);
```

I'm on my phone but I think it fits 80 chars.